elixir/apps/domain/priv/repo/migrations/20250830051953_add_flows_token_id_index.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Domain.Repo.Migrations.AddFlowsTokenIdIndex do
+  use Ecto.Migration
+
+  @disable_ddl_transaction true
+
+  def change do
+    create(index(:flows, [:token_id]))
```

Comment:
This migration will fail the deployment because the other app servers will also try to run it. I believe you need `create_if_not_exists` and the concurrently flag here.

```suggestion
    create_if_not_exists(index(:flows, [:token_id], concurrently: true))
```